CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# [Unreleased](https://github.com/seedco/StackViewController/compare/0.4.0...HEAD)
```

Comment:
Would it be possible to have Changelog under [release](https://github.com/seedco/StackViewController/releases)? So it's all in one place

Reply:
The intent for this is to be a document that we keep up-to-date as we make changes. When we do a release we can copy and paste the information in this file to the release notes without having to go back into the git history to figure out what we did.

Reply:
When new release created with empty description it takes all the changes from commit history. 
For me it looked like this:
<img width="1023" alt="screen shot 2017-11-23 at 1 02 06 pm" src="https://user-images.githubusercontent.com/1641795/33188416-961b0226-d04e-11e7-8497-13cd944ad651.png">

Before it was formatting this way though: Reference to the PR

<img width="1060" alt="screen shot 2017-11-23 at 1 03 24 pm" src="https://user-images.githubusercontent.com/1641795/33188436-b82555f6-d04e-11e7-8556-c036fb30dc2e.png">